Dockerfile

```diff
@@ -80,6 +80,11 @@ RUN mkdir /opt/convert3d && \
     curl -fsSL --retry 5 https://sourceforge.net/projects/c3d/files/c3d/Experimental/c3d-1.4.0-Linux-gcc64.tar.gz/download \
     | tar -xz -C /opt/convert3d --strip-components 1
 
+# Julia for MEDIC
+FROM downloader as julia
+RUN curl -fsSL https://install.julialang.org | sh -s -- --yes --default-channel 1.9.4 && \
+    mkdir -p /opt/julia/ && cp -r /root/.julia/juliaup/*/* /opt/julia/
```

Comment:
Can we add it to env.yml and install it with mamba instead?

Reply:
Should be done now